test/testsupport/testdata/TdEssttpDatesBodies.scala

```diff
@@ -18,18 +18,33 @@ package testsupport.testdata
 
 object TdEssttpDatesBodies {
 
-  def extremeDates(): String =
+  def extremeDatesRequest(): String =
+    """{
+      |
+      |""".stripMargin
```

Comment:
is this supposed to be empty JSON? Isn't there a closing bracket missing?

Reply:
woops, forgot to delete this

Reply:
Deleted